engine/service.go

```diff
@@ -2,43 +2,54 @@ package engine
 
 import (
 	"fmt"
+	"os"
 	"strconv"
 	"sync"
 
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
 
 	"github.com/longhorn/longhorn-engine-launcher/rpc"
+	emutil "github.com/longhorn/longhorn-engine-launcher/util"
```

Comment:
what's wrong with the name `util`?

Reply:
I see from a previous code. Normally we call the util in the same package `util`.